src/apps/chifra/pkg/base/address.go

```diff
@@ -52,13 +51,25 @@ func (a *Address) IsZero() bool {
 	return v == "0x0000000000000000000000000000000000000000"
 }
 
+func (h *Address) ToCommon() common.Address {
+	return common.BytesToAddress(h.Bytes())
+}
+
+func (h *Address) FromCommon(c *common.Address) Address {
+	return BytesToAddress(c.Bytes())
+}
```

Comment:
We might want to think about using Go conventional names for setters and getters. It would be `Common()` to get common.Address and `SetCommon` instead of `FromCommon`: https://go.dev/doc/effective_go#Getters